docs/community/contribute/grammar-formatting-and-style.md

```diff
@@ -28,6 +28,47 @@ In a list of three or more items, follow each item except the last with a comma
 | One, two, three, and four.    | One, two, three and four.    |
 | Henry, Elizabeth, and George. | Henry, Elizabeth and George. |
 
+
+### References to Filecoin
+
+As a proper noun, the name "Filecoin" (capitalized) should be used only to refer to the overarching project, to the protocol, or to the project's canonical network:
+
+> Filecoin [the project] has attracted contributors from around the globe!
+
+> Filecoin [the protocol] rewards contributions of data storage instead of computation!
+
+> Filecoin [the network] is currently storing 50 PiB of data!
+
+The name can also be used as an adjective: 
+
+> The Filecoin ecosystem is thriving!
+
+> I love contributing to Filecoin documentation!
+
+When referring to the token used as Filecoin's currency, the unit symbol, `FIL`, is preferred; it is alternatively denoted by the Unicode symbol for an integral with a double stroke (⨎). These symbols can be used as suffixes or prefixes, as seen fit:
+
+- Code prefix: **100 FIL**.
+- Code suffix: **FIL 100**.
+- Symbol prefix: **⨎100**.
+- Symbol suffix: **100 ⨎**.
+
+The singular form of the common noun "filecoin" is also sometimes used for this purpose - this form should not be used as an adjective.
```

Comment:
I find this usage (lowercase filecoin for the token) confusing. I believe it's also less common. If we don't have examples where this usage is important let's eliminate altogether for simplicity.

Reply:
Roger that!